Ouranosinc/xscen
# add scripting utils

### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #xyz
- [ ] (If applicable) Documentation has been added / updated (for bug fixes / features).
- [x] (If applicable) Tests have been added.
- [x] This PR does not seem to break the templates.
- [x] HISTORY.rst has been updated (with summary of main changes).
  - [x] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added.

### What kind of change does this PR introduce?

* `save_and_update`: nothing new, just putting things that exists all together so that I only have one line at the end of each tasks.
** I removed the "move" part of that function, because we want to encourage Ouranos users to write everything in one place and than use `move_and_delete`.
* `move_and_delete`: In a workflow, when a simulation is done, you can save what you want to keep (final, maybe regridded and log file) and delete the workdir.
* added catutils to the API for the docs

### Does this PR introduce a breaking change?
no

### Other information:
* how do we do tests for io stuff ?
* I am thinking of using these function in template 1, but I am not sure. It might be better to see that save and update are different functions when you are learning xscen? Is `move_and_delete` too advanced for basic workflow ?
